crates/subspace-runtime-primitives/src/lib.rs

```diff
@@ -91,6 +91,8 @@ pub mod opaque {
     use sp_runtime::traits::{BlakeTwo256, Block as BlockT, Header as HeaderT};
     use sp_runtime::{generic, DigestItem, OpaqueExtrinsic};
     use sp_std::prelude::*;
+    #[cfg(not(feature = "std"))]
```

Comment:
doesn't `sp_std` already take care of this?

Reply:
Predule has it indeed. I had a strange compilation error when trying to test just `sc-consensus-subspace` and this fixed one of the errors, but others remained, so I abandoned trying to fix it, but left this. Reverted now.